ngraph/test/frontend/shared/src/basic_api.cpp

```diff
@@ -180,3 +180,15 @@ TEST_P(FrontEndBasicTest, DISABLED_testInputModel_setPartialShape)
 
     // TODO: not clear now
 }
+
+TEST_P(FrontEndBasicTest, testInputModel_setElementType)
```

Comment:
This is 'shared' test across all potential frontends. Do we really have such requirement for any frontend? I think no, some test models in future can be already i32, some test models may have only one 'Place', some frontends may handle this properly.
I'd recommend to change this test according to API expectations, e.g. that after 'setElementType', call something like 'getElementType' and verify it. Or 'setElementType' to some specific model for all places, and verify that conversion works well